src/journal/admin.py

```diff
@@ -12,16 +12,17 @@ class IssueAdmin(admin.ModelAdmin):
     list_display = ('volume', 'issue', 'issue_title', 'journal', 'issue_type')
     list_filter = ('journal', 'issue_type')
     search_fields = ('issue_title',)
-    filter_horizontal = ('articles', 'guest_editors')
+    filter_horizontal = ('articles', 'editors')
```

Comment:
So the admin app doesn't use `related_name`s?

Reply:
I've removed this because filter_horizontal doesn't work with m2m's that have in intermediate table anyway.

Reply:
:+1: